web/Views/Home/Index.cshtml

```diff
@@ -3,30 +3,79 @@
     ViewData["Title"] = "Home Page";
 }
 
-<div class="text-center">
-    <h1 class="display-4">Bricks & Hearts</h1>
-    <p>Welcome to Bricks & Hearts.</p>
-    <p>Under construction for <a href="https://www.changeahead.org.uk/">Change Ahead</a></p>
-    <p style="max-width: 800px; margin-left: auto; margin-right: auto">This is a (work in progress) portal for landlords to manage properties they would like to make available to disadvantaged tenants looking for social housing.</p>
-    @{
-        if (Model.IsLoggedIn)
-        {
-            <p>
-                Logged in as @Model.UserName
-            </p>
-            if (Model.IsRegisteredAsLandlord)
-            {
-                <a class="btn btn-primary" asp-controller="Landlord" asp-action="MyProfile">Go to profile</a>
-            }
-            else
-            {
-                <a class="btn btn-success" asp-controller="Landlord" asp-action="RegisterGet">Register as a Landlord</a>
-            }
-            <a class="btn btn-success" asp-controller="Admin" asp-action="AdminDashboard">Admin Dashboard</a>
-        }
-        else
-        {
-            <a class="btn btn-primary" href="@Url.Action("Google", "Login")">Login with Google</a>
-        }
-    }
-</div>
\ No newline at end of file
+
+@if (Model.IsLoggedIn)
+{
+    <div class="container-fluid text-center">
+        <div class="row mb-4">
+            <h2>Welcome to</h2>
+            <statistics class="display-4">Bricks & Hearts</statistics>

```

Comment:
What's the `<statistics>` tag about, never seen that before?

I'd expect something like

```suggestion
            <h2>Welcome to <span class="display-4">Bricks & Hearts</span></h2>
```
So a screen reader can figure out it's all part of the header